packages/ui/components/ui/tx/actions-views/delegator-vote.tsx

```diff
@@ -0,0 +1,127 @@
+import { ViewBox } from '../viewbox';
+import { ValueViewComponent } from '../../value';
+import { getAddress } from '@penumbra-zone/getters/note-view';
+import { ActionDetails } from './action-details';
+import {
+  DelegatorVoteBody,
+  DelegatorVoteView,
+  Vote,
+  Vote_Vote,
+} from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/component/governance/v1/governance_pb';
+import { getDelegatorVoteBody } from '@penumbra-zone/getters/delegator-vote-view';
+import { Amount } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/num/v1/num_pb.js';
+import {
+  Metadata,
+  ValueView,
+} from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/asset/v1/asset_pb.js';
+import { AddressViewComponent } from '../../address-view';
+import { base64ToUint8Array } from '@penumbra-zone/types/base64';
+
+// TODO: This is sad, but at the moment, we aren't provided the metadata to have a rich display.
+//       Given the high-priority of getting action view support, this is added.
+//       We should properly implement ValueViews into the protos for DelegatorVote action view and delete this code.
```

Comment:
@grod220 retro reviewing.

regarding the to-do, can we retrieve this information from the `TransactionViewComponent` similarly to how we handle it for the `SwapViewComponent`? refer to this [example](https://github.com/penumbra-zone/web/pull/1419/files#diff-c4407fc8843a60c40b2a2bb104fad82b6a50e11ef1807380dff00f9fc5e6c4e6R16-R37).

Reply:
I'm not quite seeing what you mean. The issue is that DelegatorVoteView is missing ValueView's. Until we update those protos, we won't have what we need for this view.